acestep/training/lora_utils.py

```
@@ -368,48 +380,55 @@ def load_training_checkpoint(
+        try:
+            device_str = None
+            if device is not None:
+                # load_file expects a device string like "cpu" or "cuda:0"
+                device_str = str(device)
+            training_state_tensors = load_file(state_path_safe, device=device_str)  # type: ignore[arg-type]
+
+            # Expect scalar tensors for epoch/global_step if present
+            if "epoch" in training_state_tensors:
+                try:
+                    result["epoch"] = int(training_state_tensors["epoch"].item())
+                except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)